llvm/lib/DebugInfo/GSYM/GsymCreator.cpp

```diff
@@ -379,6 +385,12 @@ uint32_t GsymCreator::insertString(StringRef S, bool Copy) {
   return StrOff;
 }
 
+StringRef GsymCreator::getString(uint32_t offset) {
```

Comment:
```suggestion
StringRef GsymCreator::getString(uint32_t Offset) {
```

Reply:
> If you use `StringOffsetMap->at(offset)` then it will abort if the key doesn't exist. However you don't get to pass a custom error message.

Would rather have a message for the assert here.